datadog_checks_dev/tests/tooling/manifest_validator/test_v2_validator.py

```diff
@@ -49,24 +49,6 @@ def setup_route():
     set_root(current_root)
 
 
-@mock.patch(
-    'datadog_checks.dev.tooling.utils.read_metadata_rows', return_value=input_constants.ORACLE_METADATA_CSV_EXAMPLE
-)
-def test_manifest_v2_all_pass(_, setup_route):
-    """
-    Run a valid manifest through all V2 validators
-    """
-    validators = get_all_validators(False, "2.0.0")
-    for validator in validators:
-        # Currently skipping SchemaValidator because of no context object and config
-        if isinstance(validator, v2_validators.SchemaValidator):
-            continue
-
-        validator.validate('active_directory', JSONDict(input_constants.V2_VALID_MANIFEST), False)
```

Comment:
I'm a little confused by the this test definition. `V2_VALID_MANIFEST` is a manifest for `oracle`, but the check_name being passed into the validate call is `active_directory`. I'm a bit surprised this would be passing _before_ we migrated active_directory to V2??